package.json

```diff
@@ -58,5 +58,10 @@
   "engines": {
     "node": "^18.20.1",
     "pnpm": "^9.6.0"
+  },
+  "pnpm": {
+    "patchedDependencies": {
```

Comment:
ooc does this have to be in the workspace root for it to work? or could this be in `packages/explorer/package.json`

Reply:
looks like it's not possible to scope it down to a package, pnpm complains if I do so :/